force-app/main/default/classes/UTIL_BatchJobService.cls

```diff
@@ -154,6 +154,21 @@ public without sharing class UTIL_BatchJobService {
         return getCompletedDaysBetween(batchJob.CompletedDate, batchJob.Status);
     }
 
+    /***
+    * @description Determines number of days passed since the batch job has completed
+    * @param batchId Batch job Id
+    * @return Integer
+    */
+    public Integer getCompletedDaysByBatchId(Id batchId) {
+        AsyncApexJob batchJob = getAsyncApexJob(batchId);
+
+        if (batchJob == null) {
+            return MAX_DAYS_EXCEEDED;
```

Comment:
I get why we're returning a large number here (anything greater than 7 would work, right?), but I find it confusing that this method returns `MAX_DAYS_EXCEEDED` for `null`, while `getCompletedDaysByClassName` returns 0 🤷‍♂️

Reply:
change to return -1 when batch job does not exist.